### PR TITLE
feat: components

### PR DESCRIPTION
**Pull request by owlot**
- Split the HTML into components.
- Using YAML files for content blocks.
- Updated content texts as per https://docs.google.com/document/d/10ZMUM_QaFlArR7js-1oUXkwgClSo9LAGKvyvoUtgzjI

Deployed on demo environment:
https://www.dmo.flyarrow.io/

### CODE REVIEW

**Review by ServiceDog — CHANGES_REQUESTED** [review]

File: src/components/Slide/Conops.vue [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<template>
+    <div id="conops" class="container p-5 conops">
+        <div class="row text-center">
+            <div>
+                <p>Want a more extensive read on our software structure?</p>
+                <a href="#" class="btn btn-secondary">Download the CONOPS</a>
```
[the remark this answers is not in the document]
Comment (owlot) [reply src/components/Slide/Conops.vue:6]: We can skip the slide for now yeah. Don't know how far we are with the CONOPS.

File: src/components/Slide/Infrastructure.vue
[the remark this answers is not in the document]
Comment (owlot) [reply src/components/Slide/Infrastructure.vue:1]: @miss-queenNL will go over this version and do the last design checks. I'll ensure this is also noted for her to make it clear it's draggable.

File: src/content/discover.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          Packages are scanned at every step of the journey. On pickup, a bill of lading is generated for you and
+          your customers' records. Cut your delivery time with direct aerial routes and reach more customers in
+          rural areas. Optionally deploy your own cargo drones for greater control over your deliveries.
+      </p>
+  - id: medical-support
+    title: Medical Support
+    subtitle: Shave valuable minutes, save precious lives.
+    content:
+      <p>
+        Leave behind the constraints of traditional transportation methods and embrace increased speed and reliability in delivering critical care. Organ delivery, vaccines, blood packets, etc.
```
[the remark this answers is not in the document]
Comment (owlot) [reply src/content/discover.yaml]: fixed: 496bb9f0c35b8a63239139904045c8e8762b8f37

File: src/components/Slide/Conops.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (owlot) [reply src/components/Slide/Conops.vue:6]: Commented out in 496bb9f0c35b8a63239139904045c8e8762b8f37

**Review by ServiceDog — APPROVED** [review]

**Pull request merged**